internal/makefile/REUSE.toml

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: SAP SE
+# SPDX-License-Identifier: Apache-2.0
+
+version = 1
+
+[[annotations]]
+path = [
+  ".github/CODEOWNERS",
+  ".github/renovate.json",
+  ".gitignore",
+  ".license-scan-overrides.jsonl",
+  ".license-scan-rules.json",
```

Comment:
If we have this, why do we need `license-scan-overrides.jsonl.license` and `license-scan-rules.json.license` in this same directory? I don't see these being used.